sechub-sereco/src/test/java/com/daimler/sechub/sereco/test/AssertVulnerabilities.java

```diff
@@ -175,6 +190,46 @@ public VulnerabilityFinder withType(String type) {
             search.setType(type);
             return this;
         }
+        
+        public FindCodeCallStackBuilder withCodeLocation(String location, int line, int column) {
+            return new FindCodeCallStackBuilder(location,line,column);
+        }
+        
+        public class FindCodeCallStackBuilder{
+            
+            private SerecoCodeCallStackElement currentCallStackElement;
+
+            public FindCodeCallStackBuilder(String location, int line, int column) {
+                currentCallStackElement = createCallStackElement(location, line, column);
+                
+                search.setCode(currentCallStackElement);
+            }
+            
+            public FindCodeCallStackBuilder calling(String location, int line, int column) {
+                SerecoCodeCallStackElement childCallstackElement =createCallStackElement(location, line, column);
+
+                currentCallStackElement.setCalls(childCallstackElement);
+                currentCallStackElement=childCallstackElement;
```

Comment:
Please add spaces around the assignment operator.

```suggestion
                currentCallStackElement = childCallstackElement;
```

Reply:
done: formatted source code

Remark: please do my a favor and just mention that the code was not formatted by our source formatter. This would be time and effort saving.

Mabye something like this:

> Please apply source formatter to: AssertVulnerabilities.java , ... ,...  ?

Reply:
I will do this in the future.